src/services/dca/config.ts

```diff
@@ -4,4 +4,4 @@ import { Address as ViemAddress } from 'viem';
 export const DCA_HUB_ADDRESS: ViemAddress = '0xA5AdC5484f9997fBF7D405b9AA62A7d88883C345';
 export const DCA_PERMISSION_MANAGER_ADDRESS: ViemAddress = '0x20bdAE1413659f47416f769a4B27044946bc9923';
 export const COMPANION_ADDRESS: ViemAddress = '0xDf0dbc66f85979a1d54671c4D9e439F306Be27EE';
-export const COMPANION_SWAPPER_ADDRESS: ViemAddress = PERMIT2_ADAPTER_ADDRESS;
+export const COMPANION_SWAPPER_ADDRESS: ViemAddress = PERMIT2_ADAPTER_ADDRESS(0);
```

Comment:
I would prefer we hardcode the address, since using a `0` chainId might lead to confussion in the future
```suggestion
export const COMPANION_SWAPPER_ADDRESS: ViemAddress = '0xA70C8401C058B6198e1cb085091DE13498CEc0dC';
```